modules/nw-sctp-verifying.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+// * networking/using-sctp.adoc
+
+[id="nw-sctp-verifying_{context}"]
+= Verifying Stream Control Transmission Protocol (SCTP) works on your cluster
+
+You can verify that SCTP is working on a cluster by creating a Pod with an application that listens for SCTP traffic, associating it with a Service, and then connecting to the exposed service.
+
+.Prerequisites
+
+* Access to the Internet from the cluster to install the `lksctp-tools` package.
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
+* Access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Create a Pod that runs an SCTP listener:
+
+.. Create a file named `sctp-server.yaml` that defines a Pod using the following YAML:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  namespace: sctpdemo
+  name: sctpserver
+  labels:
+    app: sctpserver
+spec:
+  containers:
+    - name: sctpserver
+      image: fedora:31
+      command: ["/bin/sh", "-c"]
+      args:
+       ["dnf install -y lksctp-tools && sctp_test -H localhost -P 30100 -l"]
+      ports:
+        - containerPort: 30100
+          name: sctpserver
+          protocol: SCTP
+----
+
+.. Create the Pod by running the following command:
++
+----
+$ create -f sctp-server.yaml
```

Comment:
it should be `oc create -f sctp-server.yaml`

Reply:
Agree